core/.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+  ],
+  plugins: ["@typescript-eslint"],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: "module",
+  },
+  overrides: [
+    {
+      files: ["*.ts", "*.tsx"],
```

Comment:
I think you could just add `*.js` to ignorePatterns and you shouldn't need this?

Reply:
This gets added by `@typescript:recommended` but not `@typescript:eslint-recommended` So I need to manually add this override otherwise I have to run eslint with `eslint . --ext ts,tsx`